code/game/gamemodes/gameticker.dm

```diff
@@ -67,6 +67,12 @@ var/round_start_time = 0
 		initialtpass = 1
 		votetimer()
 
+/datum/controller/gameticker/proc/exptimer()
+	spawn(0)

```

Comment:
The contents of this proc can be moved into a roundstart hook in `job_exp.dm`.

This spawn should match with the sleep below, or players will get 5 minutes of exp the moment a round starts.

5 should probably be in a define in `job_exp.dm`, after you've moved this there.